docs/src/literate-tutorials/ns_vs_diffeq.jl

```diff
@@ -1,3 +1,10 @@
+# Putting this flag to false reproduces the figure shown in the example #src
+# We check for laminar flow development in the CI                       #src
+if @isdefined is_ci    #hide
+    IS_CI = false      #hide
+else                   #hide
+    IS_CI = true       #hide
```

Comment:
Is there a reason why we cannot use is_ci directly?

Reply:
I think so, since `is_ci` is not defined if a user executes it locally

Reply:
should this go to false?